website/docs/examples/controller_header/README.md

```diff
@@ -0,0 +1,165 @@
+---
+id: controller-info
+title: Controller 101
```

Comment:
This location is OK to get the content out, but we should separate out conceptual overviews and API reference from the examples section. I'll make a note for now in our docs organization topic.

Reply:
#214